docs/index.md

```diff
@@ -35,6 +35,57 @@ non-public development, custom hardware handled by vendor
   source hardware and open instruction set architecture. Feel free to check the
   website about recent vPub plans or subscribe to newsletter.
 
+## Newcommers
+
+If you are new to Dasharo, this section it to help you get started using Dasharo
+firmware and possibly contributing to making it better.
+
+### Coreboot
+
+If you have no prior experience with coreboot, it's highly recommended to visit
+OpenSecurityTraining2 and finish these courses:
+* [Arch4031](https://p.ost2.fyi/courses/course-v1:OpenSecurityTraining2+Arch4031_x86-64_RV_coreboot+2021_v1/about),
+* [Arch4021](https://p.ost2.fyi/courses/course-v1:OpenSecurityTraining2+4021_Intro_UEFI+2022_v1/about)
+OpenSecurityTraining2 provides other great courses related to firmware, which
+are all freely avialable [here](https://p.ost2.fyi/courses).
+
+### Docker
+
+Coreboot needs a specific toolchain to be built, hence why it is usually built
+inside of a Docker. In case of Dasharo, all images are build using Docker, so
+you will make sure it works properly on your system.
+
+Follow these two links:
+* [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+* [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+
+### GitHub
+
+All code review and all issues related to Dasharo are resolved on
+[GitHub](https://github.com/). An account there is nessesary to contribute and
+report issues. All Dasharo repositories can be found
+[here](https://github.com/Dasharo).
+
+When you have an account on GitHub go ahead and
+[configure an SSH key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
+It's also highly recommended to configure a GPG key before contributing
+antthing. Instructions to do that can be found
+[here](https://docs.github.com/en/authentication/managing-commit-signature-verification/adding-a-gpg-key-to-your-github-account).
+With that your account should be ready to contribute to Dasharo. Here are some
+additional links to help:
+* [Dasharo code structure](https://docs.dasharo.com/dev-proc/source-code-structure/)
+* [Creating a pull pequest from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)
+* [Pull request eviews](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews)
+
+### Dasharo Matrix Space
+
+[Matrix](https://matrix.org/) as a communicator used at Dasharo. If you want
+quick answers it's best to join our matrix space and talk to us there.
+
+https://matrix.to/#/#dasharo:matrix.org
```

Comment:
lets make it clickable link not something to copy&paste

Reply:
Done, also added link to matrix clients which might be helpful if someone never used matrix before.